Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.h

```diff
@@ -203,6 +208,16 @@ class ModelProcessModelPlayerProxy final
     // For testing
     bool m_unloadDelayDisabledForTesting { false };
     static uint64_t gObjectCountForTesting;
+
+#if ENABLE(MODEL_ELEMENT_IMMERSIVE)
+    bool m_immersivePresentation { false };
+    WebCore::LayoutSize m_inlineFrameSize { };
```

Comment:
why not m_layoutSize? :) (we only "layout" on the page so the inline is kinda implied)